cfgov/unprocessed/apps/ccdb-search/webpack-config.js

```diff
@@ -2,70 +2,72 @@
    Settings for webpack JavaScript bundling system.
    ========================================================================== */
 
-   const TerserPlugin = require( 'terser-webpack-plugin' );
+const TerserPlugin = require( 'terser-webpack-plugin' );
 
-   /* Set warnings to true to show linter-style warnings.
+/* Set warnings to true to show linter-style warnings.
       Set mangle to false and beautify to true to debug the output code. */
```

Comment:
Trivial: align the comment to its new indentation above.

Reply:
Fixed in 2e529be4eb50be9c927b4aeb046a8eee6bddb9df